src/components/Header.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import md5 from 'crypto-js/md5';
+import { connect } from 'react-redux';
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+    this.getGravatar = this.getGravatar.bind(this);
+  }
+
+  getGravatar() {
+    const { email } = this.props;
+    const hash = md5(email).toString();
+    const urlImage = `https://www.gravatar.com/avatar/${md5(hash).toString()}?s=200`;
```

Comment:
Oi, @felipedclc!

Reparei que você fez a _hash_ do e-mail na constante `hash` e, ao definir a string, você fez uma _hash_ sobre a string da _hash_ do e-mail. Pensei que funcionaria só com a _hash_ do e-mail. Desse jeito:

```suggestion
    const urlImage = `https://www.gravatar.com/avatar/${hash}?s=200`;
```

Posso estar enganado, mas, se você testou e deu certo, pode ignorar a sugestão.

Caso isso tenha passado percebido, deixo já a sugestão para mudar o código.